siliconcompiler/_metadata.py

```diff
@@ -1,6 +1,10 @@
 # Version number following semver standard.
 version = '0.9.6'
 
+# Default server address for remote runs, if unspecified.
+default_server = 'https://server.siliconcompiler.com'
+default_server_name = default_server.replace('https://', '').strip('/')
```

Comment:
Maybe we can make a utility function that strips the url instead of having this in the metadata. I think it's okay to have the real URL, but the derivative seems like that should not live here.

Reply:
Putting it here means less code in client.py and the configure app, and we don't really need the logic to strip off these leading/trailing characters anywhere else. It's only there so that it will pick up on "server.siliconcompiler.com" and "https://server.siliconcompiler.com/", both of which would be valid entries in the credentials file.

Reply:
okay, we'll address this in a followon PR